internal/blockchain/ethereum/ethereum.go

```diff
@@ -205,7 +207,6 @@ func (e *Ethereum) handleBatchPinEvent(ctx context.Context, msgJSON fftypes.JSON
 	}
 
 	if sBlockNumber == "" ||
-		sTransactionIndex == "" ||
```

Comment:
Should we validate the presence of `txIndex` and `logIndex`?

Reply:
The code as written will fall back to zero for these if not present (consistently across all paths in eth and fabric).

Would you prefer I move all to rejecting the event if they're not explicitly set to a numeric value?

Reply:
Hmm no, I think zero is OK. Thanks for the clarification.